uftrace.c

```diff
@@ -148,6 +149,7 @@ static struct argp_option uftrace_options[] = {
 	{ "task-newline", OPT_task_newline, 0, 0, "Interleave a newline when task is changed" },
 	{ "chrome", OPT_chrome_trace, 0, 0, "Dump recorded data in chrome trace format" },
 	{ "diff", OPT_diff, "DATA", 0, "Report differences" },
+	{ "format", OPT_format, "csv", 0, "Use csv output format" },
```

Comment:
We may provide more format methods in addition to "csv" so what about this?
```
{ "format", OPT_format, "FORMAT", 0, "Use FORMAT for output format" },
```

Reply:
Fixed